string topic_name   # Topic to which data is written
string message_data   # What data is to be wriiten
